Source/WebCore/rendering/FloatingObjects.h

```diff
@@ -110,6 +114,7 @@ class FloatingObject {
     unsigned m_paintsFloat : 1;
     unsigned m_isDescendant : 1;
     unsigned m_isPlaced : 1;
+    unsigned m_hasTrimmedMargins : 1;
```

Comment:
Where do you initialize this member?

Reply:
Thanks for catching this. Initialized in the constructors now

Reply:
FloatingObject seems to have multiple constructors and I am only seeing one initialization here.